app/models/ms_specimens.php

```diff
@@ -830,13 +830,19 @@ public function importIDBSpecimen($o_request, $vn_project_id, $vs_uuid) {
 						$vb_taxon_linked = false;
 						$vb_taxon_created = false;
 						$o_search = new TaxonomyNamesSearch();
-						$q_taxon_hits = $o_search->search(trim($va_specimen_info["indexTerms"]["specificepithet"]." ".$va_specimen_info["indexTerms"]["genus"]." ".$va_specimen_info["indexTerms"]["infraspecificepithet"])."*", array('sort' => 'ms_taxonomy_names.genus'));
+						$q_taxon_hits = $o_search->search(trim($va_specimen_info["indexTerms"]["genus"]." ".$va_specimen_info["indexTerms"]["specificepithet"]." ".$va_specimen_info["indexTerms"]["infraspecificepithet"])."*", array('sort' => 'ms_taxonomy_names.genus'));
```

Comment:
Maybe this would be more readable?

```
$q_taxon_hits = $o_search->search(
	trim($va_specimen_info["indexTerms"]["genus"]." ".
		$va_specimen_info["indexTerms"]["specificepithet"]." ".
		$va_specimen_info["indexTerms"]["infraspecificepithet"])."*", 
	array('sort' => 'ms_taxonomy_names.genus'));
```

Reply:
reformatted and pushed